theme_aviato/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+{
+    'name': 'Aviato',
```

Comment:
Find the intruder ;)
![Screenshot from 2021-08-30 11-58-50](https://user-images.githubusercontent.com/10338094/131324918-26bdd295-800e-4e7e-b208-75f1e9ed59f0.png)